javascript/todo/src/task.js

```diff
@@ -0,0 +1,23 @@
+import Controls from "./controls";
+
+class Task extends Controls {
+  #date;
+  #order;
+  constructor(title, discription, dueDate, priority) {
+    super(title, discription);
+    this.#date = dueDate;
+    this.#order = priority;
+  }
+  getDueDate() {
+    return console.log(this.#date);
+  }
+  setDueDate(dueDate) {
+    this.#date = dueDate;
+  }
+  getPriority() {
+    return console.log(this.#order);
+  }
+  setPriority(pirority) {
```

Comment:
setPriority method needs more validation to make sure users enter a number 
recommended : 
number should be between 1 - 10